frontend/src/components/pages/VolunteerShifts/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Container } from "@chakra-ui/react";
+import React, { useContext, useEffect, useState } from "react";
+import { Status } from "../../../types/AuthTypes";
+import PendingPage from "./PendingPage";
+import VolunteerAPIClient from "../../../APIClients/VolunteerAPIClient";
+import AuthContext from "../../../contexts/AuthContext";
+
+const VolunteerShiftsPage = () => {
+  const [volunteerStatus, setVolunteerStatus] = useState<Status>();
+  const { authenticatedUser } = useContext(AuthContext);
+  
+  const getScheduleData = async () => {
+    const volunteerResponse = await VolunteerAPIClient.getVolunteerByUserId(authenticatedUser!.id);
+    setVolunteerStatus(volunteerResponse.status);
+  };
+
+  useEffect(() => {
+    getScheduleData();
```

Comment:
Can we rename this to `getVolunteerData()`

Reply:
whoops, thanks for catching that